pkg/utils/kube/ctl.go

```diff
@@ -102,10 +103,14 @@ func (k *KubectlCmd) filterAPIResources(config *rest.Config, preferred bool, res
 	return apiResIfs, nil
 }
 
-// isSupportedVerb returns whether or not a APIResource supports a specific verb
+// isSupportedVerb returns whether or not a APIResource supports a specific verb.
+// The verb will be matched case-insensitive.
 func isSupportedVerb(apiResource *metav1.APIResource, verb string) bool {
+	if verb == "" || verb == "*" {
```

Comment:
Is this something that can happen? As far as I can tell, all instances of the argument are hard-coded as values others than these two.

Reply:
Good question. This is intended to allow the caller to just say "I don't care about the support verbs, just resolve it". It might be a valid use-case for some.